### PR TITLE
Feature/newsletter and drift changes

### LINKED ISSUE #153: CloudSlang website only focused on DevOps technology - Proposal review
Hello,

My 2 cents about the current home page website. It is based on customer feedbacks. 

My opinion is that we should not focus CloudSlang website (and mostly homepage) on DevOps use cases and technologies only. It's very restrictive and It sounds for new people like CloudSlang usage is only for around Docker, Jenkins, CoreOS to help new DevOps team to automate.

Unfortunately, it sounds as another tool for Docker ecosystem only, which is not the case. It would be great to have it more attractive like best-in-class orchestration technology for YAML textual design.

We should announce a process-based orchestrator engine for runbook automation uses cases (Incident Remediation, Fullfillment, User management, disaster recovery, ...). Same story than HP OO around the use cases.

We should also show the textual design through YAML DSL based value (standard de facto), Python and JAVA native capabilities with enterprise grade engine.

What do you think about this proposal ?

Thanks for comments and review.

```
Automate your DevOps use cases using ready-made workflows
```

```
What is CloudSlang?

CloudSlang is an open source tool for orchestrating popular DevOps technologies, such as Docker and CoreOS in an agentless manner. Use ready-made workflows or define your own custom ones. CloudSlang workflows are reusable, shareable and easy to understand.
```

### PR DESCRIPTION
**Pull request by andreivasiletruta**
I am opening the pull request in order to join the latest website changes with the master.
a. drift added
b. several pages and sub-sections of several pages changed

fixes #153

### CODE REVIEW

File: app/views/about.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <div ng-include src="'views/navbar.html'"></div>
 </div>
 <div class="container">
-    <div class="row row-centered" style="margin-bottom: 150px;">
+    <div class="row row-centered" style="margin-bottom: 50px;">
```
Comment (AndraLuana) [inline app/views/about.html]: can't you add the margin in row-centered? or maybe add a new css class, because maybe in the future you want to customize it more... also it's a good practice to use classes/ids not inline styling

File: app/styles/main.css [hunk 2]
```diff
@@ -256,6 +256,48 @@ a:active, a:focus {
...
+    }
+    .navbar-toggle {
+        display: block;
+    }
+    .navbar-collapse {
+        border-top: 1px solid transparent;
+        box-shadow: inset 0 1px 0 rgba(255,255,255,0.1);
+    }
+    .navbar-collapse.collapse {
+        display: none!important;
```
Comment (AndraLuana) [inline app/styles/main.css]: is it mandatory to use !important? it may be dangerous to override styles using it

Comment (andreivasiletruta) [inline app/styles/main.css]: It seemed like based on the tests. I would keep it for the moment as It did not reveal anything problematic so far 👍

File: app/views/about.html (hunk 1 above)
Comment (andreivasiletruta) [inline app/views/about.html]: I totally agree. however I tried to avoid any refactoring based actions. It should be definitely done. maybe we can do it in a separate pull request.
